### DISCUSSION TITLE
Clarifications in USB to UART connection docs

### DISCUSSION THREAD

**Discussion by kiddyfurby** [devbis/z03mmc#123] [General]
Would it be possible to mark which pin (P7 or P14) is RX / TX in `assets/Mi_SWS_Connection.jpg`?
If that's too much effort, simple text note in README would do!

**Comment by devbis** [comment]
Maybe you have a nice picture with the wiring? p14 is both RX and TX, as it is just an emulation of singe-wire protocol. 
p7 on the picture is an indicator with led+resistor and is not required for flashing. Actually, the picture was taken from atc1441 repo, so I don't have good photos of it.
